src/utils/ethereum.js

```diff
@@ -13,20 +14,25 @@ export const callL1Contract = async (contract, method, args = []) => {
   return response;
 };
 
-export const sendL1Transaction = async (
+export const sendL1Transaction = (
   contract,
   method,
   args = [],
   options = {},
   callback = () => {}
 ) => {
-  const [response, error] = await promiseHandler(
-    contract.methods?.[method](...args).send(options, callback)
-  );
-  if (error) {
-    return Promise.reject(error);
-  }
-  return response;
+  return new Promise((resolve, reject) => {
```

Comment:
Is'nt it possible to do it with `promiseHandler`?

Reply:
`proimiseHndler` is to handle an existing promise (instead of try-catch syntax) no to returning a new promise